### PR TITLE
config: Add option to turn off progress bars

### PR DESCRIPTION
**Pull request by iarna**

### CODE REVIEW

File: doc/misc/npm-config.md [hunk 1]
```diff
@@ -608,6 +608,16 @@ Set to true to run in "production" mode.
    local `npm install` without any arguments.
 2. Set the NODE_ENV="production" for lifecycle scripts.
 
+### progress
+
+* Default: true
+* Type: Boolean or `"always"`
```
Comment (thefourtheye) [inline doc/misc/npm-config.md]: No information about `always`?

Comment (iarna) [inline doc/misc/npm-config.md]: oops, good catch, "always" isn't an option

**Comment by zkat** [comment]
:sheep:

**Comment by thefourtheye** [comment]
lgtm :+1:

**Comment by iarna** [comment]
Landed in `npm@3.2.0`

**Pull request closed without merging**